rdm/xml_util.py

```diff
@@ -0,0 +1,63 @@
+from xml.etree import ElementTree
+
+DISABLED_PREFIX = 'DISABLED_'
```

Comment:
I can't remember, but did we need the ability to exclude disabled tests?  If possible, ideally we would rely on the test runner to exclude tests (and indicate that they were excluded via the "status" or via some other way).  Relying on the test runner seems better than adding our own mechanism.

Reply:
The information in the test output seems to be sufficient for us to deduce a status of 'skipped'. So I think this is handled.